<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPinout</class>
 <widget class="QWidget" name="SerialPinout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>128</width>
    <height>20</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QCheckBox" name="enaBox"/>
   </item>
   <item>
    <widget class="Line" name="line4">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="CTSLabel">
     <property name="text">
      <string>CTS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="DSRLabel">
     <property name="text">
      <string>DSR</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line2">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="DCDLabel">
     <property name="text">
      <string>DCD</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line3">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="RILabel">
     <property name="text">
      <string>RI</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
